pkg/kubelet/images/pullmanager/image_pull_manager_test.go

```diff
@@ -435,6 +435,7 @@ func TestFileBasedImagePullManager_MustAttemptImagePull(t *testing.T) {
 		pullingFiles       []string
 		expectedPullRecord *kubeletconfiginternal.ImagePulledRecord
 		want               bool
+		wantPodCred        bool
```

Comment:
nit: `wantPodCredsRequested` would be a more fitting name

Reply:
Yes, but then there would be a lot of spurious diffs due to `go fmt` whitespace changes. I prefer not to pollute `git blame` if I can help it. But if it's okay then sure I'll change it

Reply:
The PR went one more size up :|

Reply:
Just an idea, wouldn't it be better to disregard whitespace changes when computing PR size?